README.md

```diff
@@ -173,14 +190,19 @@ Please see the [wiki](https://github.com/neovim/nvim-lspconfig/wiki) for additio
 In order for neovim to launch certain executables on Windows, it must append `.cmd` to the command name. To work around this, manually append `.cmd` to the entry `cmd` in a given plugin's setup{} call.
 
 ## Contributions
+
 If you are missing a language server on the list in [CONFIG.md](CONFIG.md), contributing
 a new configuration for it would be appreciated. You can follow these steps:
 
 1. Read [CONTRIBUTING.md](CONTRIBUTING.md).
+
```

Comment:
this makes the bullet lists too spread out. It looks bad.

Reply:
no, i only did align content, please, take a look at the screenshots below.

## before

![image](https://user-images.githubusercontent.com/61933599/130359188-7be06bba-aff5-4029-9bd8-3ea24f7b8645.png)

## after

![image](https://user-images.githubusercontent.com/61933599/130359153-fbc2b7d0-78a5-4379-b86f-34040504684b.png)

not only did i align the list, i've merged it in a whole block. you can test it by looking in the html rendered code. i could capture `after` block via standard firefox screenshot utility easily, while w/o improvements, i should select the whole area manually, because it's a united block